### DISCUSSION TITLE
Error in nextTick: "TypeError: rootState.repeaters.repeaters[repeater].map is not a function"

### DISCUSSION THREAD

**Discussion by michaeljcoyne** [area17/twill#1416] [General]
Hi I am trying to add a repeater to one of my modules forms.

When I update I get this error.

`Error in nextTick: "TypeError: rootState.repeaters.repeaters[repeater].map is not a function"`

Any ideas?

Thanks

**Comment by ifox** [comment]
Hi @michaeljcoyne,

I don't think I've seen this error before. Can you share a bit more about what you did in your codebase to add the repeater field?
